pages/docs/guides/validate.md

```diff
@@ -0,0 +1,138 @@
+---
+title: "Validate AsyncAPI documents"
+description: In this guide, you'll learn multiple ways to validate AsyncAPI documents.
+weight: 120
+---
+
+## Introduction
+This guide teaches multiple ways to validate AsyncAPI documents. 
+
+## Create AsyncAPI document
+To validate an AsyncAPI document, you must first create one. You can create one using the [AsyncAPI CLI](https://github.com/asyncapi/cli#installation).
+
+Generate a sample `asyncapi.yaml` file (AsyncAPI document) with the following CLI command: 
+
+```
+asyncapi new --example=default-example.yaml --no-tty
```

Comment:
```suggestion
asyncapi new --example=tutorial.yml --no-tty
```

please use this one in all tutorials from now on

Reply:
Noted! @derberg